openshift-ci/resources/model-registry-DSCInitialization.yaml

```diff
@@ -17,15 +17,3 @@ spec:
   monitoring:
     managementState: Managed
     namespace: opendatahub
-  serviceMesh:
```

Comment:
What does this mean, you are not installing the ServiceMesh anymore?

Reply:
No, not at all.  Service mesh is still installed via the operator and the DSC.  The DSCI shouldn't have it as that information will put into the DSCI file once service-mesh is installed.